DTDL/publication_public.json

```diff
@@ -0,0 +1,28 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:publication:public;1",
+  "@type": "Interface",
+  "extends": ["dtmi:io:tributech:publication:base;1"],
+  "displayName": "Public Publication",
+  "description": "A public publication represents the will of a Node to share data with the entire hub/ecosystem.",
+  "comment": "Only present at owning side.",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "Disabled",
```

Comment:
Soft-delete flags are often only properties on the persisted backend properties since instances won't be served anymore at APIs after deletion.
Not sure how to handle this with twins though...

Reply:
we could add them to the $metadata and store them there?

Reply:
I don't get the point. What's the issue with the property `Disabled`?